bin/sequenceserver

```diff
@@ -245,6 +245,8 @@ MSG
           puts
           puts 'Installing NCBI BLAST+.'
           puts "RUBY_PLATFORM #{RUBY_PLATFORM}"
+          puts 'Archive will be extracted in ~/.sequenceserver directory.'
```

Comment:
@vivekiitkgp How about we instead change the 'installing ncbi blast+' msg (L246) to 'Installing NCBI BLAST+ to ~/.sequenceserver/ncbi-...'?